.github/workflows/ci.yml

```diff
@@ -17,14 +20,9 @@ jobs:
           node-version: 16
           cache: yarn
       - name: Install JS dependencies
-        run: |
-          yarn
-      - name: Lint
-        run: |
-          yarn test:lint
-      - name: TypeScript
-        run: |
-          yarn test:ts
+        run: yarn --frozen-lockfile
```

Comment:
`--frozen-lockfile` is enabled by default when Yarn is run on CI.

Reply:
ahh, still holding to my v1 😄